consensus-types/light-client/bootstrap.go

```diff
@@ -0,0 +1,227 @@
+package light_client
+
+import (
+	"fmt"
+
+	fieldparams "github.com/prysmaticlabs/prysm/v5/config/fieldparams"
+	consensustypes "github.com/prysmaticlabs/prysm/v5/consensus-types"
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/interfaces"
+	"github.com/prysmaticlabs/prysm/v5/encoding/bytesutil"
+	pb "github.com/prysmaticlabs/prysm/v5/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/v5/runtime/version"
+	"google.golang.org/protobuf/proto"
+)
+
+func NewWrappedBootstrap(m proto.Message) (interfaces.LightClientBootstrap, error) {
+	if m == nil {
+		return nil, consensustypes.ErrNilObjectWrapped
+	}
+	switch t := m.(type) {
+	case *pb.LightClientBootstrapAltair:
+		return NewWrappedBootstrapAltair(t)
+	case *pb.LightClientBootstrapCapella:
+		return NewWrappedBootstrapCapella(t)
+	case *pb.LightClientBootstrapDeneb:
+		return NewWrappedBootstrapDeneb(t)
+	default:
+		return nil, fmt.Errorf("cannot construct light client bootstrap from type %T", t)
+	}
+}
+
+type bootstrapAltair struct {
+	p                          *pb.LightClientBootstrapAltair
+	header                     interfaces.LightClientHeader
+	currentSyncCommitteeBranch interfaces.LightClientSyncCommitteeBranch
```

Comment:
I added fields for object that would have to be re-created every time the getter is called. That's one additional pointer for each header and at most `6*32 bytes` for branches.